text/trace/0170-sampling-probability.md

```diff
@@ -0,0 +1,717 @@
+sampled, the adjusted count of the new span is `1/H`.
+
+Assuming `H < I` and the incoming context was not sampled, we have the
+following probability equations:
+
+```
+P(span sampled) = I
+P(parent sampled) = H
+P(span sampled | parent sampled) = 1
+P(span sampled | parent not sampled) = D
+```
+
+Using the formula above, 
+
+```
+I = 1*H + D*(1-H)
+```
+
+solve for D:
+
+```
+D = (I - H) / (1 - H)
+```
+
+Now the Sampler makes a decision with probability `D`.  Whether the
+decision is true or false, propagate `I` as the new head inclusion
+probability.  If the decision is true, begin recording a sub-rooted
+trace with adjusted count `1/I`.
+
+## Proposed specification text
+
+The following text will be added to the semantic conventions for
+recording the Sampler name and adjusted count (if known) as
+OpenTelemetry Span attributes.
+
+### Semantic conventions for Sampled spans (Proposed text)
+
+This document defines conventions for counting system-wide span events
+using sampled spans.  These conventions support accurate counting of
+system-wide events using only the fraction of spans that were
+collected in a probability sampling scheme.  With these conventions,
+consumers of OTLP Span data are able to compute approximate metrics
+about the system using only the sample data that was collected, thus
+we refer to these conventions as supporting Span-to-Metrics pipelines.
+
+The _sampling rate_, also known as _inclusion probability_, is the
+probability that a Span is included in the Sample being collected.
+Sampling rate is conveyed in a form known as _adjusted count_, which
+tells the receiver how many events in the population are represented
+by the individual Span as a result of sampling.
+
+The adjusted count of a span is defined as follows:
+
+- Adjusted count equals zero when inclusion probability equals zero
+- Adjusted count equals the mathematical inverse (i.e., reciprocal) of inclusion probability when inclusion probability is non-zero.
+
+The zero value for adjusted count can be used when recording a Span
+that was not selected by the Sampler, as a means of conveying
+exceptional events while maintaining accurate accounting.
+
+Consumers of spans carrying an adjusted count attribute are able to
+use the adjusted count of the span to increment a counter of matching
+spans.  This probabilistic counting method is will be accurate as long
+as the Sampler produces unbiased adjusted counts that are expected to
+equal true population counts.
+
+#### Probability Sampling Attributes (Proposed text)
+
+The `sampler.adjusted_count` attribute, when set, MUST equal an
+unbiased estimate of the number of representative spans in the
+population of spans in the system.
+
+The _exported count_ associated with a span is defined as either 1 or
+0, depending on whether the span is exported and thus counted.  The
+exported count is 1 if the span is exported (because it will be
+counted) and 0 if the span is not exported.
```

Comment:
I suggest removing this notion of "exported count", as it is confusing and not particularly useful - it's only used in the table where depending on the context it's always 0 or 1.

Reply:
Done. Thanks.